tsconfig.json

```diff
@@ -9,8 +9,7 @@
     "strictNullChecks": true,
     "noUnusedLocals": true,
     "pretty": true,
-    "skipLibCheck": true,
```

Comment:
@schmod Is there a reason why you removed this line?

Reply:
When set to `true`, tsc doesn't validate/check our `.d.ts` files.  This is how my original bug slipped through.

Because we're writing a bunch of our own `.d.ts` files and are semver-strict with most of our dependencies, it seems like there are very few benefits to skipping these checks.

Reply:
Ah ok. Not what I've expected. I assumed that only libraries aren't checked with `skipLibCheck: true` as the name stated.

That explains why other typing related issues weren't revealed by running `tsc`.

Thanks for clearing.